pkg/services/user/userimpl/store.go

```diff
@@ -88,8 +88,18 @@ func (ss *sqlStore) Insert(ctx context.Context, cmd *user.User) (int64, error) {
 }
 
 func (ss *sqlStore) Get(ctx context.Context, usr *user.User) (*user.User, error) {
+	ret := &user.User{}
```

Comment:
I think this is the right thing to do, but I hope no code relied on the passed in `usr` argument being changes as a result of this call.

Reply:
@Jguer @IevaVasiljeva i am not finding that we expose this in the interface for the service, nor is this being used throughout the code base when I search for references.

the interface exposes GetById, GetByEmail. https://github.com/grafana/grafana/blob/main/pkg/services/user/userimpl/user.go#L204

i might be incorrect here and seems like a stretch to delete this function, but as far as I see this is not being used 🤔

Reply:
sorry this is satisfying the store interface! nvm